### PR TITLE
Replace travis with github actions

### PR DESCRIPTION
**Pull request by lukebryant**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Architecture
+version: '3'
+
+
```
Comment (Phil-Barber) [inline docker-compose.yml:4]:
```suggestion
```

Comment (Phil-Barber) [reply docker-compose.yml:4]: don't know whether removing both blank lines is better

**Review by Phil-Barber — APPROVED** [review]
Looks correct to me! 
Definitely squash this one 😂

File: .github/workflows/action.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+on: [push, pull_request]
```
Comment (Phil-Barber) [inline .github/workflows/action.yml:1]: Oh the only thing is whether we want both here? I think just pull_request is fine

Comment (lukebryant) [reply .github/workflows/action.yml]: I changed  it so it runs on pull request, and on push if the branch is master

**Pull request merged**